frameworks/Rust/anansi/anansi-raw.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.64
+FROM rust:latest
```

Comment:
Please keep these tied down to a specific version.

Reply:
Done.